ddtrace/tracer/option_test.go

```diff
@@ -498,3 +498,14 @@ func TestWithHostname(t *testing.T) {
 		assert.Equal("hostname-middleware", c.hostname)
 	})
 }
+
+func TestWithLogStartup(t *testing.T) {
+	var c config
+	defaultValue := c.logStartup
```

Comment:
We already know that the zero-value of a boolean is false, so we can simplify this and just do:
```go
c := newConfig()
assert.False(t, c.logStartup)
WithLogStartup()(&c)
assert.True(t, c.logStartup)
```
That should do for the whole test. I don't think there's a need for documentation either. It's pretty self-explanatory.

Reply:
Thanks!